classes/Webdashboard/Utils.php

```diff
@@ -1,52 +1,28 @@
 <?php
 namespace Webdashboard;
 
+/**
+ * Utils class
+ *
+ * Collection of static methods (cache, get population coverage)
+ *
+ *
+ * @package Webdashboard
+ */
 class Utils {
-    /*
-     * getBugsFromCSV()
-     * fetches and a remote csv file generated by an advanced search in Bugzilla
-     * @var string $csv, url of the csv file
-     * @var string $full, default to false, return short or long results
-     * @return array list of bugs and their description
-     */
-    public static function getBugsFromCSV($csv, $full = false)
-    {
-        $shortBugs = $fullBugs = $temp = [];
-
-        if (($handle = fopen($csv, 'r')) !== false) {
-
-            while (($data = fgetcsv($handle, 300, ',')) !== false) {
-
-                if ($data[0] == 'bug_id') {
-                    $fields = $data;
-                    continue;
-                }
-
-                foreach ($fields as $key => $field) {
-                    $temp[$field] = $data[$key];
-                }
-
-                $fullBugs[] = $temp;
-                $shortBugs[$temp['bug_id']] = $temp['short_desc'];
-            }
-
-            fclose($handle);
-        }
-
-        return $full ? $fullBugs : $shortBugs;
-    }
-
-    /*
-     * cacheUrl()
-     * Caches a remote resource in the cache folder for 120 seconds
+    /**
+     * Cache a remote resource in the cache folder for 120 seconds
      *
-     * @var $url, remove content
-     * @var $time = 120, seconds to cache
-     * @return string
+     * @param   string  $url         Content to fetch
+     * @param   string  $time        Seconds to cache (default 120)
+     * @param   string  $cache_path  Path to cache
+     *
+     * @return  string               URI of the cached resource, or original
+     *                               source if not available
      */
-    public static function cacheUrl($url, $time = 120)
+    public static function cacheUrl($url, $time = 120, $cache_path)
```

Comment:
The idea is to not have a class method depending on something (in this case a global constant) that's not defined or available in the class itself. It makes code harder to understand, and in this case the cost is only to have an additional parameter in method calls. 

In the long term it probably makes sense to have a Cache object where the cache path is defined only once for istance.